rebasehelper/plugins/output_tools/__init__.py

```diff
@@ -51,7 +51,17 @@ def get_report_path(cls, app):
         return os.path.join(app.results_dir, REPORT + '.' + cls.EXTENSION)
 
     @classmethod
-    def prepend_results_dir_name(cls, *path_members):
+    def prepend_results_dir_name(cls, app, *path_members):
+        if app.conf.results_dir and app.conf.bugzilla_id:
```

Comment:
Same as below, this shouldn't be special-cased.

Reply:
How come? I feel like there are 4 cases with different behaviour that can occur.

1. just `--bugzilla-id` is used, we should prepend the path with `created_repo/rebase-helper-results`
2. just `--results-dir` is used, we should prepend the path with `config.results_dir/rebase-helper-results`
3. both of the above are used, we should use what was passed in `--results-dir`. The problem is when `--results-dir` is a relative path. We could obviously convert all of the paths to absolute but I think if the user passed a relative path, we should keep it as relative, if possible (hence the special case)
4. every other case, results directory is in the current working directory.

Reply:
The point of `start_dir` was to avoid special casing of `--bugzilla-id` (and other potential future options that would change working directory). There are only two cases, `--results-dir` and no `--results-dir`.